config/suppressions.xml

```diff
@@ -107,6 +107,10 @@
   <!-- until https://github.com/checkstyle/checkstyle/issues/5234 -->
   <suppress id="MatchXPathBranchContains" files="[\\/]DetailAstImplTest.java"/>
 
+  <!-- usage DefaultLogger is necessary for MetadataGeneratorUtil.java -->
+  <suppress id="ImportControlMain"
+            files="[\\/]src[\\/]main[\\/]java[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]meta[\\/]MetadataGeneratorUtil.java"/>
```

Comment:
Add the class to the import control file.

Reply:
Will create a separate PR for this after this one gets merged.